### DISCUSSION TITLE
Scanning Excel files with compare_df_cols()

### DISCUSSION THREAD

**Discussion by sfirke** [sfirke/janitor#471] [Show and tell]
I've done a lot of wrangling-a-set-of-Excel-sheets in my day but I'm not sure it ever occurred to me until now to pipe from `read_excel()` into `compare_df_cols()`.  Pretty neat to get this without creating any objects in the environment:

![image](https://user-images.githubusercontent.com/7569808/151251152-2b3392d0-b1be-4d7e-b24b-48f0ff4f2404.png)

**Comment by billdenney** [comment]
Cool idea!
